telegram_bot/bot_handlers/handle_view_statistic.py

```diff
@@ -0,0 +1,26 @@
+from database.users_quiz_progress_table import fetch_user_stat_by_topic
+from telegram_bot.bot_message_generators.statistic_menu_generator import send_user_statistic_menu
+
+
+async def handle_view_statistic_menu(call, bot, chat_id):
+    user_id = call.from_user.id
+    query_user_statistic_by_topic = await fetch_user_stat_by_topic(user_id)
+    message = ''
```

Comment:
The handler must process the data and pass it in the correct form to the generator